DVCodingStyle.md

```diff
@@ -461,6 +462,125 @@ utility macros for even more common code sequences.
         If the macro is defined in a local perspective, it must be undefined at
         the end of the file, otherwise it will pollute the global namespace.
 
+#### Typical gotchas with macro usage:
+
+While it is better to avoid macros altogether, they do make our lives easy by
+shortening pieces of repeated code, in a manner that cannot be achieved with a
+function or a task. In general, the following describes the best practices to
+avoid common gotchas with macro usage.
+
+1.  Always wrap macro arguments **and** the output as applicable, within
```

Comment:
I think this would be clearer as something like this: "Always wrap macro arguments in parentheses. If the macro evaluates to an expression, also wrap the whole expansion in parentheses."

Otherwise, the reader has to work out what "as applicable" means.